backend/config.json

```diff
@@ -85,7 +83,9 @@
         ],
         "preferred": "/api-osoc",
         "authScheme": "auth/osoc2",
-        "defaultUserId": 3
+        "defaultUserId": 3,
+        "port": 4096,
+        "pageSize": 3
```

Comment:
3 seems rather small but we can keep it at that for easier testing.

Reply:
Right, gotta change!